api/python/provisioner/params_spec.yaml

```diff
@@ -57,6 +57,13 @@ release:
   _path: release.sls
   target_build: release/target_build
 
+mgmt_network:
+  _path: cluster.sls
+  mgmt_public_ip: cluster/srvnode-0/network/mgmt/public_ip
```

Comment:
@mazinamdar what `srvnode-0` means here?

Reply:
Initially each node will be considered as srvnode-0.